tools/staticbuild/build_lib_cmake.sh

```diff
@@ -39,14 +44,18 @@ rm -rf lib; mkdir lib;
 if [[ $PLATFORM == 'linux' ]]; then
     cp -L build/libmxnet.so lib/libmxnet.so
     cp -L staticdeps/lib/libopenblas.so lib/libopenblas.so.0
-    if [[ -f /usr/lib/gcc/x86_64-linux-gnu/4.8/libgfortran.so ]]; then
-        cp -L /usr/lib/gcc/x86_64-linux-gnu/4.8/libgfortran.so lib/libgfortran.so.3
-    elif [[ -f /usr/lib/x86_64-linux-gnu/libgfortran.so.3 ]]; then
-        cp -L /usr/lib/x86_64-linux-gnu/libgfortran.so.3 lib/libgfortran.so.3
+    if [[ $ARCH == 'aarch64' ]]; then
+        cp -L /usr/lib/aarch64-linux-gnu/libgfortran.so.5 lib/libgfortran.so.5
```

Comment:
Have you tried the one-liner `cp -L $(ldd lib/libmxnet.so | grep libgfortran |  awk '{print $3}') lib/` from master branch? It may be easier to maintain

Reply:
addressed